playbooks/container/build-publish-tag.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Build and Push Container Image
+  hosts: "all"
+
+  vars:
+    docker_registry: "registry.scs.community"
+    registry_project: ""
+    container_name: "{{ zuul.project.short_name }}"
+    containerfile_path: "" # Should be set to path to containerfile inside the repo with leading /, e.g. /containerfiles
+    dockerfile: "Dockerfile" # Alternate name for the Dockerfile
```

Comment:
How does `containerfile_path` and `dockerfile` differ? `dockerfile` is not even used. Did you mean build context and `containerfile` ?

Reply:
`containerfile_path` is the path to the directory which contains the Dockerfile or Containerfile Manifest. `containerfile` is a variable to overwrite the default filename to use in that directory to build.

e.g. `containerfile_path: "/image-manifests"` would tell the job to look in the directory `image-manifests` for container manifests. If there are two Manifests called `Container1-containerfile` and `Container2-containerfile` and setting `containerfile: "Container2-containerfile"`. It would build an image for `Container2-containerfile`.

But I see that point that both options can be combined or refactor into a more intuitive way. I will give it a thought.